src/ts/components/charts/fragments/BarChart.tsx

```diff
@@ -94,7 +94,7 @@ const BarChart = ({
             data-dash-is-loading={getLoadingState(loading_state) || undefined}
             barChartProps={newProps}
             barProps={barPropsFunction}
-            valueFormatter={resolveProp(valueFormatter)}
+            valueFormatter={resolveProp(valueFormatter) || ((value: number) => value.toString())}
```

Comment:
Minor point: this will create a new function each render, which may cause the underlying component to rerender more than necessary. Better would be to create this function once as a const outside of the component function and reuse it.